src/__tests__/runner.test.js

```diff
@@ -288,6 +288,32 @@ describe('Runner', () => {
       const lastIndex = args.lastIndexOf('--reporters');
       expect(args[lastIndex + 1]).toBe(expected[1]);
     });
+
+    it('calls createProcess with --runInBand arg when provided', () => {
+      const expected = '--runInBand';
+
+      const workspace: any = {};
+      const options = {runInBand: true};
+      const sut = new Runner(workspace, options);
+      sut.start(false);
+
+      const args = (createProcess: any).mock.calls[0][1];
+      const index = args.indexOf(expected);
+      expect(index).not.toBe(-1);
+    });
+
+    it('calls createProcess with --bail arg when provided', () => {
+      const expected = '--bail';
+
+      const workspace: any = {};
+      const options = {bail: true};
+      const sut = new Runner(workspace, options);
+      sut.start(false);
+
+      const args = (createProcess: any).mock.calls[0][1];
+      const index = args.indexOf(expected);
+      expect(index).not.toBe(-1);
+    });
```

Comment:
I appreciate that you've maintained the coding style for the tests before this.

This is a non-blocking comment about the style/readability. I've been away from the repo and Jest long enough I had to look up the mock.calls type. I think some array destructuring and using the `toContain` matcher would make these two a bit more readable at a glance (for someone like me who's been absent):
```js
      const [_workspace, args] = (createProcess: any).mock.calls[0];     
      expect(args).toContain('--runInBand'); // or .toContain(expected);
```